kokudaily/sql/cust_node_report_setup.sql

```diff
@@ -26,30 +26,23 @@ INSERT
       report_month,
       pod_count,
       cpu_cores,
-      memory_bytes
+      memory_gigabytes
   )
 SELECT ''%%1$s'' AS "customer",
        rpp.provider_id AS "provider_id",
        rpp.cluster_id AS "cluster_id",
        ro.node as "node",
-       date_trunc(''month'', rp.interval_start)::date AS "report_month",
-       count(DISTINCT ro.pod) AS "pod_count",
+       date_trunc(''month'', rpp.report_period_start)::date AS "report_month",
+       0 AS "pod_count",
        max(ro.node_capacity_cpu_cores) AS "cpu_cores",
-       max(ro.node_capacity_memory_bytes) AS "memory_bytes"
-       -- starting with line item as we need the data ingestion counts
+       max(ro.node_capacity_memory_gigabytes) AS "memory_gigabytes"
 FROM   %%1$s.reporting_ocpusagelineitem_daily_summary ro
-       -- usage report has the usage bounds
-JOIN   %%1$s.reporting_ocpusagereport rp
-ON     rp.id = ro.report_id
-AND    rp.interval_start < ''%%3$s''::timestamptz  -- start must be < end bounds as end bounds is start of next month
```

Comment:
Since we're not using interval start anymore we probably still want to add a WHERE clause where we limit the `usage_start` from `reporting_ocpusagelineitem_daily_summary` using the start and end dates passed in.